languages.md

```diff
@@ -3,8 +3,8 @@
 
 | Language    | Key        | Status | Version | GitHub    | Container | Annotations |
 |-------------|------------|--------|---------|-----------|-----------|-------------|
-| Python      | python     | beta   | v0.9.0  |[✓](https://github.com/bblfsh/python-driver) | [✓](https://hub.docker.com/r/bblfsh/python-driver/) | [✓](https://github.com/bblfsh/python-driver/blob/master/ANNOTATION.md) |
-| Java        | java       | beta   | v0.6.0  |[✓](https://github.com/bblfsh/java-driver) | [✓](https://hub.docker.com/r/bblfsh/java-driver/) | [✓](https://github.com/bblfsh/java-driver/blob/master/ANNOTATION.md) |
+| Python      | python     | beta   | v1.0.0  |[✓](https://github.com/bblfsh/python-driver) | [✓](https://hub.docker.com/r/bblfsh/python-driver/) | [✓](https://github.com/bblfsh/python-driver/blob/master/ANNOTATION.md) |
+| Java        | java       | beta   | v1.0.0  |[✓](https://github.com/bblfsh/java-driver) | [✓](https://hub.docker.com/r/bblfsh/java-driver/) | [✓](https://github.com/bblfsh/java-driver/blob/master/ANNOTATION.md) |
```

Comment:
Maybe we should just remove version number from this table, since it's a headache to keep it updated and it's not too informative. It's also a bit weird to have `1.0.0` versions marked as beta =)

Reply:
I agree.